testproject/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2020.3.12f1
-m_EditorVersionWithRevision: 2020.3.12f1 (b3b2c6512326)
+m_EditorVersion: 2020.3.15f2
+m_EditorVersionWithRevision: 2020.3.15f2 (6cf78cb77498)
```

Comment:
unnecessary?

Reply:
I mean personally I think the project should always be on the latest version of the LTS :P but that is just me. which would be .18 :P I me right now with PR triggers we aren't even testing on 2020.3 which I find also wrong.

Reply:
nightly runs on various platforms and versions at least, but yeah 2020.3 baseline for PRs might be more useful, I'm not sure

Reply:
Well seeing as how its the min version for the package it should run on it :P I mean for UTP we run on 2020.3/2021.2/trunk across all desktop platforms for PRs because more often then I would like to admit stuff breaks across unity versions. I think we should at least make sure PRs work with the min version.